datadog-iam-user-handler/requirements.txt

```diff
@@ -0,0 +1,3 @@
+cloudformation-cli-python-lib==2.1.2
+git+https://github.com/datadog/datadog-api-client-python.git@750a42c58c6246d4a0a14920c438c8986a41a7bd
```

Comment:
can be removed as it is a dep of `common`

Reply:
I'm going to revisit this requirements entirely once we tag a first version of the common. I'll fix both this and the monitor resource deps once we do that.